beluga/routes.py

```diff
@@ -85,46 +89,40 @@
     if lat or lon or radius:
         if not lat or not lon or not radius:
             raise abort(400, "search area must be fully specified")
-    
+
     # Acquire the events the user requested
     with session_scope() as db_session:
-        events = db_session.query(Event)
+        events = db_session.query(
+            Event.title,
+            Event.start_time,
+            Event.end_time,
+            Event.location.ST_AsGeoJSON().label("location_json")
```

Comment:
Actually, right here. Add a column:

```
func.ST_Distance(
        Event.location,
        WKTElement('Point({} {})'.format(lat, lon))).label('distance')
```

Then below, `order_by('distance')`.

Reply:
I think keeping them sorted by start date could be more useful for users, unless the sorting is specified somewhere